packages/terra-form-select/src/Select.jsx

```diff
@@ -145,24 +148,27 @@ function Select(props) {
 
   switch (variant) {
     case Variants.COMBOBOX: {
-      const { maxSelectionCount, ...comboboxProps } = otherProps;
+      const { maxSelectionCount, isFilterStyle, ...comboboxProps } = otherProps;
```

Comment:
do we need `isFilterStyle` to be passed for all the variants...??

Reply:
I think this is to extract `isFilterStyle` from all the props i.e `otherProps`. And later to the components only component specific props are being passed i.e `...comboboxProps` for combobox.

Reply:
yes, but I am bit confused about usage of this prop. Correct me if i am wrong, why do we need extract if it is not used by these variants..? I see this prop is declared only for single-select variant

Reply:
> why do we need extract if it is not used by these variants..?

If not extracted this will be passed to the variants where it's not supported and become part of the spread operator in there local files. This would cause an unsupported and unrecognized prop to be present in the DOM.